### DISCUSSION TITLE
Problem updating existing container with the latest image.

### DISCUSSION THREAD

**Discussion by Coppage** [felddy/foundryvtt-docker#873] [Q&A]
Hi, I’m trying to update my image v11 307. When i stop my image and re-pull from latest, i’m expecting to get v11 315. Is v11 315 the latest release you’ve uploaded?  Thanks.

**Comment by felddy** [comment]
Yes. `11.315.1` is the most recent release.  
- https://github.com/felddy/foundryvtt-docker/releases
- https://github.com/felddy/foundryvtt-docker/pkgs/container/foundryvtt/164114256?tag=11.315.1

Make sure you are not inadvertently overriding the `FOUNDRY_VERSION` environment variable.  Check the logs for warnings.

**Comment by Coppage** [comment]
It was the version number and i was also pulling the image with :11 instead of :release.  I’m all good now thanks!
